symengine/series_visitor.h

```diff
@@ -220,6 +232,83 @@ class SeriesVisitor : public BaseVisitor<SeriesVisitor<Poly, Coeff, Series>> {
         x.get_arg()->accept(*this);
         p = Series::series_atanh(p, var, prec);
     }
+    void bvisit(const Csch &x) {
```

Comment:
You can remove all of these empty methods, which are handled by `bvisit(Function)` and `bvisit(Basic)`